scripts/create_perf_json.py

```diff
@@ -1934,7 +1936,7 @@ def to_perf_json(self, outdir: Path):
                 perf_json.write('\n')
 
         # Skip hybrid because event grouping does not support it well yet
-        if self.shortname not in ['ADL', 'ADLN', 'ARL', 'LNL', 'MTL']:
+        if self.shortname not in ['ADL', 'ADLN', 'ARL', 'LNL', 'MTL', 'SRF']:
```

Comment:
Please also add `GRR` If we need to skip grouping for E-core only products (these aren't hybrids).